package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "league-of-legends-api",
+  "version": "1.0.0",
+  "main": ".eslintrc.js",
+  "scripts": {
+    "dev:migrate": "./node_modules/.bin/env-cmd ./node_modules/.bin/sequelize db:migrate --env development --config ./config/sequelize.js",
+    "dev": "./node_modules/.bin/env-cmd ./node_modules/.bin/nodemon index.js",
+    "lint:eslint": "./node_modules/.bin/eslint --format codeframe .",
```

Comment:
Remember in the grading steps when I said I would use `npm run lint`? You should have a single command that runs both of these commands.